README.md

```diff
@@ -1,52 +1,57 @@
 # River - Decentralized Chat on Freenet
 
-![Screenshot of chat interface](screenshot-20241009.png)
-
-## Current Status (Jan 2025)
-
-River is currently in active development. We're working towards an alpha release by end of January 2025. Key features currently implemented:
-
-- Basic chat functionality
-- Room creation and management
-- Invitation-based membership
-- Message signing and verification
-- Web interface foundation
-
-Please note this is pre-release software - expect breaking changes and missing features. We welcome testing and feedback from early adopters!
-
 River is a decentralized group chat system built on Freenet, designed to provide a secure and
 upgradeable alternative to traditional chat platforms. It features a web-based interface built
 with [Dioxus](https://dioxuslabs.com) and a modular contract architecture using the freenet-scaffold
 framework.
 
-## Project Structure
+![Screenshot of chat interface](screenshot-20241009.png)
 
-- [common](common/): Shared code for contracts and UI
-- [ui](ui/): Web-based user interface
-- [contracts](contracts/): River chat room contract implementation
+## Current Status (Jan 2025)
 
-## Key Features
+River is currently in active development. We're working towards an alpha release by end of January 2025. Key features currently implemented:
 
 🌐 **Web-based Interface** - Modern web UI built with Dioxus for cross-platform compatibility  
 🔒 **Secure by Design** - Uses elliptic curve cryptography for authentication and signing  
 🔄 **Upgradeable** - Flexible upgrade mechanism for both UI and contracts  
 🌱 **Extensible** - Open architecture allows alternative UIs and integrations  
 📜 **Modular Contracts** - Built using freenet-scaffold for composable state management  
 📦 **Efficient Storage** - Uses CBOR serialization via [ciborium](https://crates.io/crates/ciborium)  
+✉️ **Invitation-based membership** - Requiring invitations helps combat spam
+
+Please note this is pre-release software - expect breaking changes and missing features. We welcome testing and feedback from early adopters!
 
 ## Getting Started
 
-To join a River chat room, you'll need:
-1. The room's contract address (derived from its public key)
-2. An invitation from an existing member
+River runs in your browser, and is built to work both on mobile phones and desktop computers.
+
+1. Launch River by following this link: (there is no link yet, because River isn't ready for use at this point)
```

Comment:
we shouldn't have this until it's ready - when River is ready to use they'll need to install Freenet first then follow a link to River, but we shouldn't go into detail yet because it may confuse people into thinking it's ready to try.

Reply:
I added a disclaimer at the top of the section to make it clear that it's not ready, and also updated the list with your clarification.

Let me know if you want the entire "getting started" removed! I wasn't sure what you meant in your comment.